Sample SDK docs

divyaamunugama opened the pull request:
## Purpose
> Added Docs related to integrating sample SDKs. The SDKs are 

> - React
> - Angular
> - JS

nilminiwso2 commented on en/docs/sdks/integrate-a-js-app.md:
```diff
@@ -0,0 +1,43 @@
+Download the [sample JavaScript application](https://github.com/asgardeo/asgardeo-auth-spa-sdk/releases/latest/download/asgardeo-html-js-app.zip).
+
+## Configure the sample
+1. Open the `index.html` file located at the root of the project 
```
```suggestion
1. Open the `index.html` file located at the root of the project.
```

nilminiwso2 commented on en/docs/sdks/integrate-a-js-app.md:
```diff
@@ -0,0 +1,43 @@
+Follow the steps given below to authenticate users to a sample JavaScript service provider with OpenID Connect.
+
+## Prerequisites
+- Only customer users can log in to applications. [Create a user account](../../guides/identity-lifecycles/onboard-overview) if you don't already have one.
```
We don't need a bullet point when there is only one point

nilminiwso2 commented on en/docs/sdks/integrate-a-js-app.md:
```diff
@@ -0,0 +1,43 @@
+
+{!fragments/register-a-service-provider.md!}
+
+## Basic OAuth/OpenID Connect configuration
```
this title doesn't match the others. we have to do something about that...

nilminiwso2 commented on en/docs/sdks/integrate-a-js-app.md:
```diff
@@ -0,0 +1,43 @@
+
+## Create a service provider
+
+{!fragments/register-a-service-provider.md!}
```
We need a way to incorporate the SP configs relevant to the sample we are using.

divyaamunugama commented:
Changes with commit [33fa45d](https://github.com/wso2/docs-is/pull/2879/commits/33fa45dcd9a78c3cc7abbd686188f2997a823b47)

1. Added a new folder "Start integrating apps" and added the docs inside it
2. Removed fragments and added content manually to solve this: https://github.com/wso2/docs-is/pull/2879#discussion_r834909518
3. Updated titles where necessary

divyaamunugama commented:
Changes with commit [1b56e69](https://github.com/wso2/docs-is/pull/2879/commits/1b56e696ef49bd611d944ef2c942423fb5b15754)

Added a new step and a warning to react SDK guide.
Solves issue: https://github.com/wso2/product-is/issues/13363

nilminiwso2 commented on en/docs/sdks/start-integrating-apps/integrate-a-js-app.md:
```diff
@@ -0,0 +1,53 @@
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```

nilminiwso2 commented on en/docs/sdks/start-integrating-apps/integrate-a-react-app.md:
```diff
@@ -0,0 +1,58 @@
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```

nilminiwso2 commented on en/docs/sdks/start-integrating-apps/integrate-an-angular-app.md:
```diff
@@ -0,0 +1,53 @@
+
+## Configure a service provider
+
+1. On the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
```suggestion
1. On the WSO2 Identity Server management console (`https://<IS_HOST>:<PORT>/carbon`) go to **Main** > **Identity** > **Service Providers**.
```
